airbyte-integrations/connectors/source-freshdesk/source_freshdesk/streams.py

```diff
@@ -0,0 +1,273 @@
+#
+# Copyright (c) 2021 Airbyte, Inc., all rights reserved.
+#
+
+
+from abc import ABC
+from typing import Any, Iterable, List, Mapping, MutableMapping, Optional
+from urllib import parse
+
+import pendulum
+import requests
+import re
+from airbyte_cdk.entrypoint import logger  # FIXME (Eugene K): use standard logger
+from airbyte_cdk.models import SyncMode
+from airbyte_cdk.sources.streams.http import HttpStream
+from airbyte_cdk.sources.utils.sentry import AirbyteSentry
+from requests.auth import AuthBase
+from source_freshdesk.utils import CallCredit
+
+
+LINK_REGEX = re.compile(r'<(.*?)>;\s*rel="next"')
```

Comment:
This can be a class attribute of `FreshdeskStream` if it's not used elsewhere.

Reply:
Sure, updated in [abcae41](https://github.com/airbytehq/airbyte/pull/12334/commits/abcae4135576a3541a76979a09babf3a955c6b8f).